libnmstate/nm/device.py

```diff
@@ -49,6 +49,93 @@ def delete(dev):
         con_profile.delete()
 
 
+def reapply(dev, connection_profile):
+    """Reapply the given device or remote connection profile."""
```

Comment:
IMHO a device cannot be reapplied... Also the doc string does not really contain more information than the function name itself...